git-ftp

```diff
@@ -811,6 +819,40 @@ handle_remote_protocol_options() {
 	[ "$CURL_INSECURE" != "0" ] && REMOTE_CMD_OPTIONS="$REMOTE_CMD_OPTIONS -k"
 }
 
+extract_folder_snapshot() {
+    local fulladdress=$1
+    SNAPSHOT_DIR="$(basename "$fulladdress")"
+}
+
+save_credential() {
+    git config git-ftp.user $REMOTE_USER
+    git config git-ftp.url $URL
+    git config git-ftp.password $REMOTE_PASSWD
+}
+
+init_new_repository() {
+    
+    [ -z "$URL" ] && print_error_and_die "Error: give a url to snapshot." "$ERROR_USAGE"
+    [ -z $SNAPSHOT_DIR ] && extract_folder_snapshot $URL   # empty string, "clone" into folder with the same name as in the server
+    
+    if [ -d "$SNAPSHOT_DIR" ]; then    # if directory already exists (including . ) check for emptiness
+        if [ "$(ls -A $SNAPSHOT_DIR)" ]; then
+            print_error_and_die "Directorty $SNAPSHOT_DIR is not empty. In order to initialize a git repository the folder has to be empty." "$ERROR_NOT_EMPTY_DIR"
+        fi
+    else
+        mkdir $SNAPSHOT_DIR 2>/dev/null || print_error_and_die "Error creating directory $SNAPSHOT_DIR. Aborting." "$ERROR_MKDIR"
```

Comment:
What was your reason to add `2>/dev/null`? Couldn't there be useful information why the command failed?